.storybook/main.js

```diff
@@ -1,4 +1,27 @@
+const path = require("path");
+
+// Required by webpack.config.js
+process.env.INFURA_API_KEY = "dummy";
+process.env.NETWORK = "local";
+
+// your app's webpack.config.js
+const custom = require("../webpack.config.js");
+
 module.exports = {
-  stories: ["../src/**/*.stories.js", "../src/**/*.stories.tsx"],
+  stories: ["../src/**/*.stories.tsx"],
   addons: ["@storybook/addon-actions", "@storybook/addon-links"],
+  webpackFinal: (config) => {
```

Comment:
Soyou do have to use this composition function?  `.storybook/webpack.config.js` doesn't work after all?

Reply:
It works, but there are more things from the project's webpack.config needed.
To avoid duplication, I prefer to do it like this instead.
Turns out what I was missing were the plugins. I could probably just include it in the storybook's webpack, but I don't want to maintain 2 webpack configs.

Reply:
Now that you say this, I recall having to fine tune webpack too in one personal project where I used story book. I had some issues with module resolution, or something, I can't really remember

Reply:
> I had some issues with module resolution

Yep, had that too. Fixed in another PR.
Now that I'm just copying the app's config, no more issues (hopefully)